bridge/client/app/_components/ktb-webhook-settings/ktb-webhook-settings.component.ts

```diff
@@ -29,6 +31,25 @@ export class KtbWebhookSettingsComponent implements OnInit {
     emptyText:
       'No secrets can be found.<p>Secrets can be configured under the menu entry "Secrets" in the Uniform.</p>',
   };
+  public sendFinishedOverlayConfig: DtOverlayConfig = {
+    pinnable: true,
+    originY: 'center',
+  };
+  public _eventType: string | undefined;
+
+  @Input()
+  set eventType(eventType: string | undefined) {
+    if (this._eventType != eventType) {
+      this._eventType = eventType;
+      if (this._eventType !== 'triggered') {
```

Comment:
How should `*` be handled? (If the webhook subscribes to `triggered`, `started`, and `finished`)

Reply:
According to @johannes-b `*` should be handled like `started` and `finished` and the checkbox should be disabled in that case.
@johannes-b can you verify this again?